app/controllers/users_controller.rb

```diff
@@ -654,15 +663,18 @@ def set_user
   end
 
   def user_scope
-    if helpers.moderator?
+    if helpers.at_least_moderator?
```

Comment:
Same comment re helpers

Reply:
This one is done as well - seems safe enough to switch